packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -0,0 +1,95 @@
+/** @jsx withSlots */
+import { useRef, useEffect, useMemo } from 'react';
+import { Animated, Easing } from 'react-native';
+import { Svg, Path } from 'react-native-svg';
+import { compose, mergeProps, withSlots, UseSlots, buildUseStyling } from '@fluentui-react-native/framework';
+import { activityIndicatorName, ActivityIndicatorProps, ActivityIndicatorType } from './ActivityIndicator.types';
+import { diameterSizeMap, lineThicknessSizeMap, stylingSettings } from './ActivityIndicator.styling';
+
+// Can also just put this within the compose, would that be better?
+const getActivityIndicatorPath = (diameter: number, width: number, color: string) => {
+  const start = {
+    x: width / 2,
+    y: diameter / 2,
+  };
+  const innerRadius = diameter / 2 - width / 2;
+  const path = `M${start.x} ${start.y} a${innerRadius} ${innerRadius} 0 1 1 ${innerRadius} ${innerRadius}`;
+
+  return <Path d={path} stroke={color} strokeWidth={width} strokeLinecap="round" />;
+};
+
+export const AnimatedSvg = Animated.createAnimatedComponent(Svg);
+const useStyling = buildUseStyling(stylingSettings);
+export const ActivityIndicator = compose<ActivityIndicatorType>({
+  displayName: activityIndicatorName,
+  ...stylingSettings,
+  slots: {
+    root: AnimatedSvg,
+  },
+  render: (props: ActivityIndicatorProps, useSlots: UseSlots<ActivityIndicatorType>) => {
+    const Slots = useSlots(props);
+    const tokens = useStyling(props).root;
+
+    const memoizedActivityIndicatorData = useMemo(
+      () => ({
+        activityIndicatorColor: props.activityIndicatorColor ? props.activityIndicatorColor : tokens['activityIndicatorColor'],
+        size: props.size ? diameterSizeMap[props.size] : diameterSizeMap[tokens['size']],
+        lineThickness: props.lineThickness
+          ? lineThicknessSizeMap[props.lineThickness]
+          : props.size
+          ? lineThicknessSizeMap[props.size]
+          : lineThicknessSizeMap[tokens['lineThickness']],
+        animating: props.animating != undefined ? props.animating : true,
+        hidesWhenStopped: props.hidesWhenStopped != undefined ? props.hidesWhenStopped : true,
+      }),
+      [],
+    );
+
+    const spinAnimation = useRef(new Animated.Value(0)).current;
+    useEffect(() => {
+      if (memoizedActivityIndicatorData.animating) {
+        Animated.loop(
+          Animated.timing(spinAnimation, {
+            toValue: 359,
+            duration: 750,
+            useNativeDriver: false,
```

Comment:
Should this not be true? Or rather, we need to do a platform check here right?

Reply:
Whoops, thought that was true. I don't think we need to platform check because it only doesn't work for web which we aren't supporting anyways.

Reply:
web, win32, and probably windows. I think an extra platform check would be a good way to avoid a confusing bug for someone maybe months down the line) who is trying to port this component to another platform and wondering why their code isn't working :D